tests/e2e/utils/visit-admin-page-with-hash.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { join } from 'path';
+
+/**
+ * WordPress dependencies
+ */
+import { getPageError } from '@wordpress/e2e-test-utils';
+
+function createURLWithHash( WPPath, query = '', hash = '' ) {
+	const url = new URL( 'http://localhost:8890' );
+
+	url.pathname = join( url.pathname, WPPath );
+	url.search = query;
+	url.hash = hash;
+
+	return url.href;
+}
+
+/**
+ * Visits admin page with hash option; if user is not logged in then it logging in it first, then visits admin page.
+ *
+ * @param {string} adminPath String to be serialized as pathname.
+ * @param {string} query String to be serialized as query portion of URL.
+ * @param {string} hash URL hash.
+ */
+export async function visitAdminPageWithHash( adminPath, query, hash = null ) {
```

Comment:
If defaulting `hash` to `null` is still wanted I'd recommend updating the docblock to include `null`, otherwise I'd recommend defaulting to an empty string:

```suggestion
export async function visitAdminPageWithHash( adminPath, query, hash = '' ) {
```

Reply:
Updated in 92ec1f3e6